Threads backend doesn't compile with Clang on OS X

It will be great if I can get the latest information of Kokkos support for OS X; in particular Clang compilers.  The information in README is a bit outdated.

=== Message from Danny Dunlavy ====
I am using Xcode 8.2:
Apple LLVM version 8.0.0 (clang-800.0.42.1)
Target: x86_64-apple-darwin15.6.0

which does not contain cstdalign.h:

/Users/dmdunla/dev/hpda-tensors/tensor/sparten/tpl/kokkos/core/src/Threads/Kokkos_ThreadsExec.hpp:53:10: fatal error: 'cstdalign' file not found
#include 

I looked online and it is not clear whether Xcode 9 has it either. So this dependency means that Apple clang may not be an option for building.

--- a/core/src/Threads/Kokkos_ThreadsExec.hpp
+++ b/core/src/Threads/Kokkos_ThreadsExec.hpp
@@ -50,7 +50,6 @@
 #include <cstdio>
 
 #include <utility>
-#include <cstdalign>
 #include <impl/Kokkos_Spinwait.hpp>
 #include <impl/Kokkos_FunctorAdapter.hpp>
 

